README.md

```diff
@@ -23,21 +23,21 @@ You can then use Chronos:
 <?php
 require 'vendor/autoload.php';
 
-use Cake\Chronos\Chronos;
+use Cake\Chronos\DateTime;
 
-printf("Now: %s", Chronos::now());
+printf("Now: %s", DateTime::now());
 ```
```

Comment:
At least our behavior is the better / more correct one ;)
As [this guy](https://www.reddit.com/r/PHP/comments/3zlvvb/chronos_a_standalone_datetime_library_originally/cynvr8b) put it.

Do you feel like this is a too big issue?